aiida/engine/processes/control.py

```diff
@@ -218,6 +218,7 @@ def _perform_actions(
     _resolve_futures(futures, infinitive, present, wait, timeout)
 
 
+#pylint: disable=too-many-branches
```

Comment:
This is now actually disabling for all of the rest of the file. You should move it next to the function definition, e.g.
```python
def _resolve_futures(  # pylint: disable=too-many-branches
```